multimodal-generation/workshop-sample/NovaReel/01-text-to-video.ipynb

```diff
@@ -67,37 +68,36 @@
   },
   {
    "cell_type": "markdown",
-   "id": "04b4d957",
+   "id": "3",
    "metadata": {},
    "source": [
     "#### Setting up storage\n",
     "\n",
-    "Generating a video takes some time - approximately 3-4 minutes to produce a 6 second video. To accomodate this execution time, the Bedrock Runtime introduces a new asynchronous invocation API. You start generating a video by calling the `start_async_invoke()` method on the Bedrock Runtime client. When the generation job completes, Bedrock automatically saves the generated video to an S3 bucket you specify.\n",
+    "Generating a video takes some time - approximately 3-4 minutes to produce a 6 second video. To accommodate this execution time, the Bedrock Runtime introduces a new asynchronous invocation API. You start generating a video by calling the `start_async_invoke()` method on the Bedrock Runtime client. When the generation job completes, Bedrock automatically saves the generated video to an S3 bucket you specify.\n",
```

Comment:
We've made improvements to latency. Can you update the first sentence to say:
"Generating a video takes 1 minute or longer depending on the duration you've selected."

Reply:
Done - see new commits.